src/main/java/com/ays/assignment/model/mapper/AssignmentSaveRequestToAssignmentMapper.java

```diff
@@ -39,6 +39,10 @@ default AssignmentEntity mapForSaving(AssignmentSaveRequest saveRequest, String
                 .description(saveRequest.getDescription())
                 .status(AssignmentStatus.AVAILABLE)
                 .build();
+
+        assignmentEntity.setPoint(saveRequest.getLatitude(), saveRequest.getLongitude());
+
+        return assignmentEntity;
     }
```

Comment:
Bu kısımda nesneyi oluşturup ardından setPoint metodunu çağırıp sonra return etmek yerine;

return AssignmentEntity.builder()
....
.build()
.setPoint(saveRequest.getLatitude(), saveRequest.getLongitude());

Şeklinde bir kullanımla daha sade bir yapı oluşturulabilir.

Reply:
`build `sonra onu denemiştim return `void `olduğundan böyle kullandım.

Reply:
@moaydogdu söylediğin gibi kullanınca void döndüğü için hata verir.

@Rapter1990 @moaydogdu Aşağıdaki gibi builder pattern içerisindeki metodu override edersek örnekteki gibi kullanabiliriz.

1. `AssignmentEntity` sınıfının en alt kısmına aşağıdaki kod bloğu eklenmeli.
```
   ...
   
   public abstract static class AssignmentEntityBuilder<C extends AssignmentEntity, B extends AssignmentEntityBuilder<C, B>> extends BaseEntity.BaseEntityBuilder<C, B> {
        public B point(final Double latitude, final Double longitude) {
            this.point = AysLocationUtil.generatePoint(latitude, longitude);
            return this.self();
      }
   }
   
   ...
```

2. Artık aşağıdaki gibi kullanılabilir;

```
return AssignmentEntity.builder()
   .id(AysRandomUtil.generateUUID())
   .institutionId(institutionId)
   .description(saveRequest.getDescription())
   .firstName(saveRequest.getFirstName())
   .lastName(saveRequest.getLastName())
   .lineNumber(saveRequest.getPhoneNumber().getLineNumber())
   .countryCode(saveRequest.getPhoneNumber().getCountryCode())
   .point(saveRequest.getLatitude(), saveRequest.getLongitude())
   .status(AssignmentStatus.AVAILABLE)
   .build();
```

Reply:
@agitrubard @moaydogdu 

Onu kullanınca `AssignmentToAssignmentEntityMapper` 'da şu kısımda sorun oluşur.

```
@Mapping(target = "point", expression = "java(mapToPoint(assignment.getLatitude(), assignment.getLongitude()))")
    AssignmentEntity map(Assignment assignment);
```

Reply:
@Rapter1990 Aslında 2 söylediğin de olmayacak, sadece return kısmında 2. Maddede yazdığımı kullanman yeterli olacak